R/jfa-methods.R

```diff
@@ -804,6 +804,31 @@ plot.jfaEvaluation <- function(x, type = c("estimates", "posterior"), ...) {
       stop(paste0('plot(..., type = "posterior") not supported for method = "', x[["method"]], '" without "prior = TRUE"'))
     }
     p <- plot.jfaPlanning(x, ...)
+  } else if (type == "sequential") {
+    if (is.null(x[["data"]])) {
+      stop(paste0('plot(..., type = "sequential") not supported when "data" is not supplied'))
+    }
+    if (is.null(x[["prior"]])) {
+      stop(paste0('plot(..., type = "sequential") not supported without "prior = TRUE"'))
+    }
+    if (is.null(x[["posterior"]][["hypotheses"]])) {
+      stop(paste0('plot(..., type = "sequential") not supported when "materiality" is not supplied'))
+    }
+    bf <- c()
+    if (x[["method"]] == "hypergeometric") {
```

Comment:
Just always give the n.units from the result object, it can also be non-null if the method is not hypergeometric